File: OP2/OP2lab2/C++/Laba2/Laba2/Laba2.cpp
#include "Header.h"
#include <iostream>
#include <string>
int main()
{
    string name1 = "file1.txt";
    string name2 = "file2.txt";
    int que;
    cout << "====>If you want to clear file enter 1: ";
    cin >> que;
    enter_f(name1, que);
    cout << "====>File 1:\n";
    ou(name1);
    char date[15];
    cout << "====>Today`s date (dd.mm.yyyy): ";
    cin >> date;
    fil(name1, name2, date);
    cout << "====>File 2:\n";
    ou(name2);
    cout << "====>Products produced in the last 10 days:\n";
    ou_sp(name1, date);
    system("pause");
    return 0;
}

File: OP2/OP2lab2/Python/Laba2/Laba2/Laba2.py
from Header import *
import pickle

que=input("====>If you want to clear file enter 1: ")
enter_f("file1.txt", que)
print("====>File 1:")
ou("file1.txt")
date=str(input("====>Today`s date (dd.mm.yyyy): "))
fil("file1.txt", "file2.txt", date)
print("====>File 2:")
ou("file2.txt")
print("====>Products produced in the last 10 days:")
ou_sp("file1.txt", date)